src/Main.tsx

```diff
@@ -76,10 +78,16 @@ export default function Main() {
     FiraCode_400Regular,
   })
 
-  const { user } = useAuth()
+  const { user, loading } = useAuth()
+
+  useEffect(() => {
+    if (hasPoppinsLoaded && hasDidactLoaded && hasFiraCodeLoaded && !loading) {
+      SplashScren.hideAsync()
+    }
+  }, [hasDidactLoaded, hasFiraCodeLoaded, hasPoppinsLoaded, loading])
```

Comment:
as it depends on so many fields, I think that the effect can be removed

Reply:
make sense :)